packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -75,7 +75,7 @@ function DropZone(props: SpectrumDropZoneProps, ref: DOMRef<HTMLDivElement>) {
 }
 
 /**
- * A dropzone is an area into which one or multiple objects can be dragged and dropped.
+ * A drop zone is an area into which one or multiple objects can be dragged and dropped.
```

Comment:
Couple of other places in the repo where we'll want to do the same change here for consistency (e.g. isFilled prop description)